index.src.html

```diff
@@ -759,9 +840,7 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
   <pre class="idl">
     typedef (FormData or URLSearchParams) CredentialBodyType;
 
-    [Constructor(PasswordCredentialData data),
-     Constructor(HTMLFormElement form),
```

Comment:
Ideally, we'd deprecate them quickly in Chrome, and other folks would never implement them. Let's see if we can do that before adding a note.